docs/index.md

```diff
@@ -495,6 +496,8 @@ The interface now can be registered and used as usual from fibers:
 {% include_snippet usage ./comsat-jooq/src/test/java/co/paralleluniverse/fibers/jooq/JooqContextTest.java %}
 ~~~
 
+As with versions 3.7 and above, jOOQ requires Java 8, Comsat provides two separate jOOQ integrations: The legacy, Java 7 and below, `comsat-jooq` and `comsat-jooq-java8` which supports using jOOQ with Java 8.   
```

Comment:
I'd change it into:

Starting with versions 3.7 jOOQ requires Java 8 and Comsat provides two separate jOOQ integrations: `comsat-jooq` supports Java 6+ and jOOQ 3.6.x while `comsat-jooq-java8` supports jOOQ 3.8.x with Java 8.